shared/js/abp-preprocessed.es6.js

```diff
@@ -125,11 +125,11 @@ function updateLists () {
 }
 
 // Make sure the list updater runs on start up
-settings.ready().then(() => abpLists.updateLists())
+settings.ready().then(() => updateLists())
 
 chrome.alarms.onAlarm.addListener(alarm => {
     if (alarm.name === 'updateLists') {
-        settings.ready().then(() => abpLists.updateLists())
+        settings.ready().then(() => updateLists())
```

Comment:
Originally I wanted to consolidate all the `chrome` events before applying Browserify, hence why I added the `abpLists` thing.

I'm not doing that anymore, which is why I'm removing it...